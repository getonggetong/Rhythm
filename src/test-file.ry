track_foo()
{
  def c;
  c = [[A0.16,A1.16,A2.16],A3,A4.16,R.8,A2];
  return c;
}

track_foo2()
{
  def c;
  c = [[A0.16,A1.16,A2.16],A3,A4.16,R.8,A4];
  return c;
}
