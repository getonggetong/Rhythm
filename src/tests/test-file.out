RYM file created successfully
RYM file created successfully